test/util.test.js

```diff
@@ -0,0 +1,18 @@
+const test = require('tape')
+
+const { bufferToURIData, URIDataToBuffer } = require('../util')
+
+test('util (ssb-uri methods)', (t) => {
+  const uriData = '8LFJ/YVGqcypcXXf0KlHXJsB/0vC9ynu/DxJKX8jlC0='
+    .replace(/\//g, '_')
+    .replace(/\+/g, '-')
```

Comment:
To reduce the amount of code in tests (the `replace`), I suggest we just copy paste an example from SSB URI spec, e.g.

```js
const uriData = 'g3hPVPDEO1Aj_uPl0-J2NlhFB2bbFLIHlty-YuqFZ3w='
```

Reply:
If you want to do that feel free to. (for me is below threshold of that I would spend time on)